spring-batch-neo4j/README.md

```diff
@@ -7,25 +7,100 @@ This extension contains an `ItemReader` and `ItemWriter` implementations for [Ne
 The `Neo4jItemReader` can be configured as follows:
 
 ```java
-SessionFactory sessionFactory = ...
-Neo4jItemReader<String> itemReader = new Neo4jItemReaderBuilder<String>()
-        .sessionFactory(sessionFactory)
-        .name("itemReader")
-        .targetType(String.class)
-        .startStatement("n=node(*)")
-        .orderByStatement("n.age")
-        .matchStatement("n -- m")
-        .whereStatement("has(n.name)")
-        .returnStatement("m")
+Neo4jItemReader<User> reader = new Neo4jItemReaderBuilder<User>()
+        .neo4jTemplate(neo4jTemplate)
+        .name("userReader")
+        .statement(Cypher.match(userNode).returning(userNode))
+        .targetType(User.class)
         .pageSize(50)
         .build();
 ```
 
 The `Neo4jItemWriter` can be configured as follows:
 
 ```java
-SessionFactory sessionFactory = ...
-Neo4jItemWriter<String> writer = new Neo4jItemWriterBuilder<String>()
-        .sessionFactory(sessionFactory)
+Neo4jItemWriter<User> writer = new Neo4jItemWriterBuilder<User>()
+        .neo4jTemplate(neo4jTemplate)
+        .neo4jDriver(driver)
+        .neo4jMappingContext(mappingContext)
         .build();
+```
+
+## Minimal Spring Boot example
+
+With a Spring Boot application containing the additional dependencies `spring-boot-starter-neo4j` and `spring-batch-neo4j`,
```

Comment:
That should go together with the following sentence.

x, y, z are needed and b must be excluded. See the following gradle for a minimal example,